.circleci/config.yml

```diff
@@ -134,6 +134,25 @@ jobs:
                 name: remove transient test fixtures
                 command: ./.circleci/scripts/pantheon/09-cleanup-fixtures
 
+    visual_regression_test:
+        <<: *defaults
+        docker:
+            - image: ataylorme/docker-node-advanced-wordpress-on-pantheon:backstop-js
```

Comment:
The only reason for a custom image here instead of using `backstopjs/backstopjs:latest` is the need for [`jq`](https://stedolan.github.io/jq/).